src/plugins/data/common/search/expressions/esql.ts

```diff
@@ -315,7 +315,14 @@ export const getEsqlFn = ({ getStartDependencies }: EsqlFnArguments) => {
             (body.all_columns ?? body.columns)?.map(({ name, type }) => ({
               id: name,
               name,
-              meta: { type: esFieldTypeToKibanaFieldType(type), esType: type },
+              meta: {
```

Comment:
Being not familiar with this are of the code, and Lukas being on PTO, I wonder, how can this be tested in the DataDiscovery world?  thx

Reply:
Go to discover, switch to esql mode and enter a query which contains `| WHERE @timestamp >= ?_tstart AND @timestamp <= ?_tend | STATS ....`, save this to a new dashboard
now change a query to add a new field, remove the auto added dimension, manually add a new dimension, or change an existing one ... the field list is empty on main, but contains correct fields on this branch